packages/salesforcedx-vscode-lightning/src/commands/quickpick/quickpick.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { paramCase } from 'change-case';
+import opn = require('opn');
+import { Uri, window, workspace } from 'vscode';
+import { LanguageClient } from 'vscode-languageclient';
+import { COMPONENT_LIBRARY_BUNDLE_LINK } from '../../constants';
+import { LwcNode } from '../../views/lwc-node';
+import { TagItem } from './tag-item';
+
+export function createQuickOpenCommand(client: LanguageClient) {
```

Comment:
It would be good to add telemetry to track the language server performance e.g. https://github.com/forcedotcom/salesforcedx-vscode/blob/79f718244dd0712836898ce5a51b659e9127f370/packages/salesforcedx-vscode-apex/src/index.ts#L77